src/assets/icons/broken_glass_blue.svg

```diff
@@ -0,0 +1 @@
+<svg id="blue_svg_2px" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 96 96"><defs><style>.cls-1{fill:#00008f;}</style></defs><title>broken_glass</title><path class="cls-1" d="M92.81,34,60.92,40.26l.9-8.88L74.48,16.14a1,1,0,0,0-1.54-1.28L58.45,32.3l-6.88-8.25L53.28,9.11a1,1,0,0,0-2-.23l-1.14,10-4.2,1-3.75-1.64-1.81-7.3a1,1,0,0,0-1.94.48l3.08,12.46-3.41,3.44L21.68,20l-9.44-8.52A1,1,0,1,0,10.9,13l29,26.18-9.33,5.66H26.77L16,40.37h0L3.38,35.16A1,1,0,0,0,2.62,37l12.21,5,10,24.5L5.54,84.09a1,1,0,1,0,1.35,1.48L31.53,63.15l8.41,3.24L30,86.56a1,1,0,0,0,1.79.89L43.48,63.73l7.22-8.21,2.76,11.14a1,1,0,0,0,.41.59L79.58,84.58a1,1,0,0,0,1.12-1.66L66.12,73.09l-6.28-11,.59-1.19,9.22-.93L72,61.79a1,1,0,1,0,1.24-1.57L70.6,58.12h0l-6.73-5.29L62,47.13l4.5-5.46L87.19,52.14a1,1,0,1,0,.9-1.79L69,40.71,93.19,36a1,1,0,0,0-.38-2Zm-35.65-.15-5.63,6.78L50.09,37,51.26,26.8ZM17.42,43.13l8.77,3.63a1,1,0,0,0,.38.08h4.29a1,1,0,0,0,.52-.15l1.72-1,4.07,9.67-10.77,9.8Zm42.66,3.15a1,1,0,0,0-.18.95l2.14,6.5a1,1,0,0,0,.33.47l5,3.94-7.71.78a1,1,0,0,0-.8.55l-1.07,2.17a1,1,0,0,0,0,.94l4.55,8L55.31,65.8,52.19,53.18a1,1,0,0,0-1.72-.42l-8.57,9.75a1,1,0,0,0-.15.22l-.92,1.86-7.64-2.94L39,56.32a1,1,0,0,0,.25-1.13L34.82,44.6l7.27-4.41a1,1,0,0,0,.15-1.6L26.43,24.32,38,29.41a1,1,0,0,0,1.12-.21l4.29-4.33a1,1,0,0,0,.26-.94l-.8-3.25,2.65,1.16a1,1,0,0,0,.4.08l.23,0,3.82-.9L48.08,37.05a1,1,0,0,0,.06.48L50.28,43a1,1,0,0,0,.77.62h.16a1,1,0,0,0,.77-.36l7.55-9.08-.74,7.26a1,1,0,0,0,1,1.1l.19,0,3.86-.76Z"/></svg>
```

Comment:
Please use `<path fill="currentColor"`. instead of inline style so that a user can override the color